### ISSUE TITLE
NightMode?

### ISSUE THREAD

**Issue by fuzgirl1** [fuzgirl1/GeekyMath#5]
This is just a random suggestion mainly because I love apps with this. Basically it could be configured under settings and it just changes the color scheme... Will be finished brb

**Comment by jodale11** [comment]
Are you suggesting that we code to a way that the user can change it to be day mode to night mode and vice versa?

**Comment by fuzgirl1** [comment]
Yes, sorry I just now realized that I never finished typing this up.

**Comment by jodale11** [comment]
Ok. I like the idea. we should probably keep this on the back burner for now, and maybe save towards the end of Febuary, and even maybe wait till state.

**Comment by fuzgirl1** [comment]
I realize that, I was throwing the idea out there. We have other things to finish like the Learn activity and redoing the main menu.

**Comment by jodale11** [comment]
Lets try focusing on the main menu portion. I feel like if we can get that portion completed, we then can proceed further into the app.

**Comment by Kitlu** [comment]
Good idea, agree with you guys here
